baselines/pFedHN/pFedHN/conf/base.yaml

```diff
@@ -0,0 +1,28 @@
+---
+dataset:
+  # dataset config
+  data: "cifar10"
+  root: "./data"
+
+model:
+  # model config
+  device: "cpu"
```

Comment:
Let's not use this for the clients. Instead is better to do this through the `client_resources` input argument to `start_simulation`. See my comment on `client.py` about this.

Reply:
Maybe it's worth having the `client_resources` as a separate node in your config. For instance like this: https://github.com/adap/flower/blob/f490dde46f2cc3b1e1786b99c2a1d8ec200907cb/baselines/fedprox/fedprox/conf/config.yaml#L12

Then pass that to `start_simulation`.

Reply:
Have made a separate function in utils .py https://github.com/achiverram28/flower/blob/main/baselines/pFedHN/pFedHN/utils.py#L25 for the device